python/lsst/ctrl/stats/records/record.py

```diff
@@ -72,14 +83,27 @@ def __init__(self, year, lines):
     def recDatetime(self):
         return parser.parse(self.timestamp)
 
+    def _utcseconds(self, dt):
+        utc_dt = dt.astimezone(tz.tzutc())
+        epoch = datetime.datetime(1970, 1, 1, 0, 0, 0)
+        epoch = epoch.replace(tzinfo=tz.tzutc())
+        return (utc_dt - epoch).total_seconds()
+
+    def _utcDatetime(self, dt):
+        timeFormat = "%Y-%m-%d %H:%M:%S"
+        utc_dt = dt.astimezone(tz.tzutc())
```

Comment:
The flake8 reports a message
"N806 variable in function should be lowercase"
in various circumstances where we end a line with ":"  and then follow with an indented line starting lowercase -- here it is like line 93  (though it can also occur after the start of a for loop).  I haven't been able to understand what the message means, so there is no particular recommended action, but I just wanted to record the observation here.